app/assets/stylesheets/spotlight/_bootstrap_overrides.scss

```diff
@@ -90,6 +90,7 @@ legend {
 }
 
 small.form-text {
+  display: block;
```

Comment:
Wouldn't it be more clear and maintainable  to add the `.d-block` class to the html?